platform/run_image.go

```diff
@@ -32,14 +32,15 @@ func GetRunImageForExport(inputs LifecycleInputs) (files.RunImageForExport, erro
 		return files.RunImageForExport{}, err
 	}
 	if len(runMD.Images) == 0 {
-		return files.RunImageForExport{}, err
+		return files.RunImageForExport{}, nil
```

Comment:
it's not clear to me why this line was changed - is it covered by the tests below?